tests/network/utils.py

```diff
@@ -304,30 +302,25 @@ def assert_service_mesh_request(expected_output, request_response):
 def assert_authentication_request(vm, service_app_name):
     # Envoy proxy IP
     expected_output = "127.0.0.6"
-    request_response = authentication_request(
-        vm=vm,
-        service=service_app_name,
-        expected_output=expected_output,
-    )
+    request_response = authentication_request(vm=vm, service=service_app_name)
     assert_service_mesh_request(expected_output=expected_output, request_response=request_response)
 
 
-def verify_console_command_output(
-    vm,
-    command,
-    expected_output,
-    timeout=TIMEOUT_1MIN,
-):
+def run_console_command(vm, command, timeout=TIMEOUT_1MIN):
     """
-    Run a list of commands inside a VM and check for expected output.
+    Run a single command through a VM console.
     """
     with console.Console(vm=vm) as vmc:
         LOGGER.info(f"Execute {command} on {vm.name}")
         try:
             vmc.sendline(command)
-            vmc.expect(expected_output, timeout=timeout)
-            return expected_output
+            vmc.expect(vmc.prompt, timeout=timeout)
+            return vmc.before
```

Comment:
It needs to return `vmc.before.decode('utf-8')`, I do not really understand how it worked as is.

Reply:
Now I understand: https://github.com/RedHatQE/openshift-virtualization-tests/blob/c2b09bcffe214e9fc9205897e58b852ba5d7f4a8/utilities/console.py#L92